nodejs/week3/calculatorRouter.js

```diff
@@ -0,0 +1,70 @@
+
+const express = require("express");
+const calculatorRouter = express.Router();
+
+// /calculator/addition?firstParam=1&secondParam=2 should respond with 3.
+
+calculatorRouter.get('/addition', (request,response) => {
+
+  console.log(request.query);
+  if (request.query.firstParam === {num1}, request.query.secondParam === {num2} ) { 
```

Comment:
I might be slow, but how does this conditional work when you are comparing a value to a variable that is assigned within the conditional? Or if I ask another way, what is this `if` doing for you?